packages/shared-components/src/components/SearchBox/styles/SearchBoxSearchButton.js

```diff
@@ -8,18 +8,21 @@ const INACTIVE_SIZE = '2em';
 export default styled.button`
   position: relative;
   display: block;
-  width: 0;
+  width: ${ACTIVE_SIZE};
+  height: ${ACTIVE_SIZE};
+  line-height: ${ACTIVE_SIZE};
```

Comment:
Do we need to change the `line-height`?

Reply:
probably not. I just made it the same as the before element.